### ISSUE TITLE
Portals are jank

### ISSUE THREAD

**Issue by 1schlick** [Specnr/RandomDimensions#1]
When I enter a portal from the Overworld, the portal in the Nether gets destroyed and replaced by a new portal 1 block higher.

Then when I enter a portal from the Nether, a new portal gets created in the Overworld about 5 blocks away from the original portal, but I spawn underground nowhere near the new portal that was created.

Also, whenever a new portal is created, the surrounding blocks get replaced by air. I lost a double chest full of goods this way.

**Comment by Specnr** [comment]
Yeah, portal creation is definitely an issue. However right now I don't have the time to fix issues with older content like this. I will definitely fix this in the future, I just can't promise it'll be very soon.

In the meantime, just be wary of where you make portals, and make sure its not near anything important
